teku/src/main/java/tech/pegasys/teku/cli/subcommand/storage/RocksDbCommand.java

```diff
@@ -49,28 +58,52 @@ public void run() {
   public int usage(
       @Mixin final BeaconNodeDataOptions beaconNodeDataOptions,
       @Mixin final Eth2NetworkOptions eth2NetworkOptions) {
+    final DatabaseVersion dbVersion = beaconNodeDataOptions.parseDatabaseVersion();
+    switch (dbVersion) {
+      case V4, V5 -> {
+        getUsage(beaconNodeDataOptions, eth2NetworkOptions, "archive");
+        getUsage(beaconNodeDataOptions, eth2NetworkOptions, "db");
+      }
+      case V6 -> getUsage(beaconNodeDataOptions, eth2NetworkOptions, "db");
+      default -> {
+        SUB_COMMAND_LOG.display("LevelDB is not supported in this command");
+        return 2;
```

Comment:
was this specific value 2 intended?

Reply:
it'll respond with a usage error...

Reply:
i double checked, no helpful usage but does at least emit 2 to $? so still helpful not to exit code 0. I might just change it to code 1 if its not doing usage.

Reply:
```
➜  teku  ✗ $TEKU --config-file tmp/minimal-altair.yaml debug-tools rocksdb usage
LevelDB is not supported in this command
➜  teku  ✗ echo $?
1
➜  teku  ✗
```